lib/inspect.c

```diff
@@ -115,7 +134,7 @@ bool foreach_peer_file(struct rpminspect *ri, const char *inspection, foreach_pe
 
         TAILQ_FOREACH(file, peer->after_files, items) {
             /* Ignore files we should be ignoring */
-            if (ignore_path(ri, inspection, file->localpath, peer->after_root)) {
+            if (ignore_path(ri, inspection, file->localpath, peer->after_root) && !has_security_checks(inspection)) {
```

Comment:
The reason I didn't is because all of the callers will be passing in the inspection parameter using a macro rather than a malloc'ed string.  I use assert() for dynamically allocated strings.